### PR TITLE
Ensure clone works without `-C` for release extraction

### PR DESCRIPTION
**Pull request by smarterclayton**
Missed a few spots when cleaning up earlier.

### CODE REVIEW

File: pkg/oc/cli/admin/release/git.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ var noSuchRepo = errors.New("location is not a git repo")
 func (g *git) exec(command ...string) (string, error) {
 	buf := &bytes.Buffer{}
 	bufErr := &bytes.Buffer{}
-	cmd := exec.Command("git", append([]string{"-C", g.path}, command...)...)
+	cmd := exec.Command("git", append([]string{"--git-dir", filepath.Join(g.path, ".git")}, command...)...)
```
Comment (wking) [inline pkg/oc/cli/admin/release/git.go]: This works, but a potentially more reliable alternative (that also supports bare repositories) would be to set [`Dir`][1] in the constructed command.

```go
cmd := exec.Command("git", command...)
cmd.Dir = g.path
glog.V(5).Infof("Executing git in %s: %v\n", cmd.Dir, cmd.Args)
```

[1]: https://golang.org/pkg/os/exec/#Cmd

Comment (smarterclayton) [reply pkg/oc/cli/admin/release/git.go]: Good point. Won't work for clone but will work for the others.

**Comment by smarterclayton** [comment]
Updated

**Comment by wking** [comment]
/lgtm

^ I may not be authorized for that, but just in case ;).

**Pull request merged**